docs/core/compatibility/networking/9.0/server-port-attribute.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "HttpClient metrics report `server.port` unconditionally"
+description: Learn about the breaking change in networking in .NET 9 where HttpClient metrics now report the `server.port` attribute unconditionally to maintain compliance with Open Telemetry standards.
+ms.date: 11/5/2024
+---
+
+# HttpClient metrics report `server.port` unconditionally
+
+When [HttpClient metrics](../../../../fundamentals/networking/telemetry/metrics.md) was implemented in .NET 8, `server.port` was introduced as a [`Conditionally Required`](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/#conditionally-required) attribute in accordance with the state of the standard at that time. Being conditionally required meant that the port was only reported if it did not match the default port of the corresponding protocol (80 for HTTP, 443 for HTTPS). However, the standard [requirement level of the attribute](https://opentelemetry.io/docs/specs/semconv/http/http-spans/#http-client) has since been changed to `Required`.
```

Comment:
The grammar sounds weird here, "HttpClient metrics" should not be singular. In my original text I wrote `When implementing HttpClient Metrics in .NET 8`.

However, since I'm not a native speaker, I'm out of ideas how to change this. Maybe you have something in mind?

Reply:
Maybe *When HttpClient metrics were implemented in .NET 8* is good?